src/main/java/dk/easv/helpers/DocumentHelper.java

```diff
@@ -0,0 +1,22 @@
+package dk.easv.helpers;
+import dk.easv.be.Document;
+import dk.easv.bll.DocumentLogic;
+
+import java.sql.SQLException;
+import java.util.List;
+
+
+public class DocumentHelper {
+    public static StringBuilder convertToString() throws SQLException {
+        List<Document> oldDocuments = DocumentLogic.showDocumentName();
+        List<String> documentNames = oldDocuments.stream()
```

Comment:
you are looping through the list twice for no reason, beware that you have to understand the code you write for the exam so steer clear of just copying stuff you dont understand